vmngclient/session.py

```diff
@@ -168,6 +172,8 @@ def __init__(
         self._session_type = SessionType.NOT_DEFINED
         self.server_name = None
         self.logger = logging.getLogger(__name__)
+        self.enable_relogin: bool = True
+        self.__second_relogin_try: bool = False
```

Comment:
The flag should be replaced by some counter which will say how many times we should try to login

Reply:
@tehAgitto AFAIK this flag was introduced to disable re-login on purpose (negative tests etc.. By default it should be enabled and advanced used can disable it after session instance is created.

Reply:
@tehAgitto Ok now i get it you want user to set max_relogin_attempts, and disabling re-login with value 0.

Reply:
@tehAgitto during discussion we found it makes no sense to do more relogin attempts in case when cookie expired.